### ISSUE TITLE
Question: need to run bitcoind and lnd locally for development?

### ISSUE THREAD

**Issue by bolatovumar** [Ride-The-Lightning/RTL#242]
I see that RTL requires `macaroonPath`, `lndConfigPath` and `bitcoindConfigPath` in `RTL.conf`. This mean that if I want to do RTL development then I have to run both bitcoind and lnd locally, correct?

I see in [this comment](https://github.com/Ride-The-Lightning/RTL/issues/95#issuecomment-477828572) that `lnd.conf` may not be required but I'm not sure of it.

**Comment by ShahanaFarooqui** [comment]
@bolatovumar only the 'macaroonPath' config is required out of above 3. 'lndConfigPath' and 'bitcoindConfigPath' can be configured if user wants to see these configurations in the UI. 

For <v0.6.4, 'lndConfigPath' was also required for authentication, if user wanted to authenticate through 'lnd.conf's `bitcoind.rpcpass`. But, we recently removed the feature to authenticate through lnd.conf in favor of multi node setup configurations.

So, in short, RTL development do NOT require any local setup. But, the admin.macaroon file has to be copied at the local machine. Hope it answers your question. Please feel free to ping us on Telegram ('https://web.telegram.org/#/im?p=@RTL_App_Community') or twitter ('https://twitter.com/RTL_App').

**Comment by ShahanaFarooqui** [comment]
@bolatovumar In some older LND versions, we had to update/add some values (rpclisten/restlisten) in 'lnd.conf' to connect from the remote machine. Hopefully, it is not needed anymore but let us know if you still are unable to connect to a remote LND node through RTL.

**Comment by bolatovumar** [comment]
@ShahanaFarooqui thanks, this clarifies it. I guess it would be nice to update this file to specify which settings are required and which are optional: https://github.com/Ride-The-Lightning/RTL/blob/master/docs/Application_configurations

**Comment by ShahanaFarooqui** [comment]
@bolatovumar, Thanks for the suggestion. We updated the documentation in `configUpgrade` branch, which will be merged tomorrow in Master and will be released as v0.6.4.

For now, you can check the updated document here: https://github.com/Ride-The-Lightning/RTL/blob/ConfigUpgrade/docs/Application_configurations

**Comment by bolatovumar** [comment]
@ShahanaFarooqui thanks
